engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -1695,14 +1695,15 @@ static void LogFrameBufferError(GLenum status)
         glGetShaderiv(shader_id, GL_COMPILE_STATUS, &status);
         if (status == 0)
         {
+            dmLogError("Unable to compile %s shader.", type == GL_VERTEX_SHADER ? "vertex" : "fragment");
 #ifndef NDEBUG
             GLint logLength;
             glGetShaderiv(shader_id, GL_INFO_LOG_LENGTH, &logLength);
             if (logLength > 0)
             {
                 GLchar *log = (GLchar *)malloc(logLength);
                 glGetShaderInfoLog(shader_id, logLength, &logLength, log);
-                dmLogError("%s\n", log);
+                dmLogError("%s", log);
```

Comment:
Here I decided to remove the extra "\n" because dmLogError adds its own new line. Plus on my GPU the error message also contains a new line and we get two empty lines. 

Purely for aesthetics.